curriculum/challenges/english/03-front-end-development-libraries/bootstrap/add-font-awesome-icons-to-all-of-our-buttons.md

```diff
@@ -78,7 +78,7 @@ assert.isTrue(
       <a href="#"><img class="img-responsive thick-green-border" src="https://cdn.freecodecamp.org/curriculum/cat-photo-app/relaxing-cat.jpg" alt="A cute orange cat lying on its back."></a>
     </div>
   </div>
-  <img src="https://cdn.freecodecamp.org/curriculum/cat-photo-app/running-cats.jpg" class="img-responsive" alt="Three kittens running towards the camera.">
+  <img src="https://cdn.freecodecamp.org/curriculum/cat-photo-app/running-cats.jpg" class="img-responsive" alt="A ginger cat walking on grass in sunlight.">
```

Comment:
I have a doubt on these ones, the image is called "running-cats" but now there is only one cat in the image

Reply:
Yes – we don't want to break the previous challenges / projects. But at the same time we don't want to negatively surprise campers. @zaira are there any images of more than one cat running that we could use instead? Or could we take one? I think Ania has two cats so maybe you could ask her to get a shot of both of them standing up (even if they aren't running.)